requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==4.2.14
+Django==4.2.14, <5
```

Comment:
I think the standard we wanted to move everything to is:
```
Django>=4.2.14,<5
```

but it's interesting to see that this works with `==` with the `,<5` as well.. I would think pip would actually fail with this.

Reply:
I will just merge this actually - as this is fine. I actually think just pinning to the exact Django version makes sense too, idk.

Reply:
It's good to be consistent. I overlooked the `>=` I'll make a new PR. No rush